frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -82,8 +86,11 @@ after(() => {
 });
 
 export const checkErrors = () =>
-  cy.window().then((win) => {
-    assert.isTrue(!win.windowError, win.windowError);
+  cy.window().then(({ windowError }) => {
+    assert.isTrue(
+      typeof windowError !== 'object' || windowError.length === 0,
```

Comment:
nit: I think this is easier to read and should work

```suggestion
      !windowError || windowError.length === 0,
```